### PR TITLE
[MM-35761] Remove references to AWS provisioning and setting AWS server variables

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR removes the deployment sidebar link and adds the developer mode setting requiremenet.

image

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-35761

### CODE REVIEW

**Comment by jfrerich** [comment]
@aaronrothschild, @justinegeffen would one of you please review this?

File: site/content/integrate/apps/quick-start-go/_index.md [hunk 1]
```diff
@@ -14,7 +14,13 @@ This quick start guide explains the basics of writing a Mattermost app. In this
...
 
-Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`. Please also ensure that `Bot Account Creation` and `OAuth 2.0 Service Provider` are enabled in the System Console.
+Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+
+In the System Console, ensure that the following are set to true:
```
Comment (justinegeffen) [inline site/content/integrate/apps/quick-start-go/_index.md:19]:
```suggestion
In the System Console, ensure that the following are set to **true**:
```

File: site/content/integrate/apps/quick-start-js/_index.md [hunk 2]
```diff
@@ -31,115 +37,115 @@ git clone https://github.com/mattermost/mattermost-plugin-apps.git
...
 make dist
 ```
 
-Then upload it via the System Console to your local Mattermost server.
+Then upload it to your local Mattermost server via the System Console.
 
 ## Building the app
 
-Start building your app by creating a directory for the code:
+Start building your app by creating a directory for the code, setup a new package and install [`node-fetch`](https://www.npmjs.com/package/node-fetch) and [`express`](https://expressjs.com/), which will be used later:
```
Comment (justinegeffen) [inline site/content/integrate/apps/quick-start-js/_index.md:48]:
```suggestion
Start building your app by creating a directory for the code, set up a new package and install [`node-fetch`](https://www.npmjs.com/package/node-fetch) and [`express`](https://expressjs.com/), which will be used later:
```

**Review by justinegeffen — APPROVED** [review]
This looks good to me! I made some UI formatting changes, but no content changes. Just want to confirm, though, is the blog post meant to be hitching a ride with this?

**Review by mickmister — APPROVED** [review]

**Review by DHaussermann — APPROVED** [review]
Changes approved LGTM!

**Pull request merged**